googletest/src/gtest.cc

```diff
@@ -119,7 +119,7 @@
 # include <sys/types.h>  // NOLINT
 #endif
 
-#include "src/gtest-internal-inl.h"
+#include "gtest-internal-inl.h"
```

Comment:
How is this related to the bazel change? Please revert and just leave the bazel related changes.